loginCompose/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why is this moved to login compose? It's a launcher icon

Reply:
removed it